pkgs/tools/archivers/peazip/default.nix

```diff
@@ -16,16 +18,20 @@
 
 stdenv.mkDerivation rec {
   pname = "peazip";
-  version = "9.4.0";
+  version = "9.5.0";
 
   src = fetchFromGitHub {
     owner = "peazip";
     repo = pname;
     rev = version;
-    hash = "sha256-to5VhuTomw33WRWrtSIMF/SD+KVXsUKmSrJ84BNatqw=";
+    hash = "sha256-3nRptI2i5t4Ds9RisxTV/MqRXEz2aNJ4hhIBh2/9Zz8=";
   };
   sourceRoot = "${src.name}/peazip-sources";
 
+  postPatch = ''
+    substituteInPlace dev/peach.pas --replace "  HSYSBIN       = 0;" "  HSYSBIN       = 2;"
```

Comment:
This makes peazip compile so that it will use the compression and archiver executables in it's PATH.